torch/csrc/autograd/python_variable.cpp

```diff
@@ -771,6 +773,79 @@ static PyObject* THPVariable_make_wrapper_subclass(
   END_HANDLE_TH_ERRORS
 }
 
+// DTensor-specific variant of make_wrapper_subclass to minimize DTensor
```

Comment:
Let's move this to a distributed folder?

Reply:
which one? we have torch/csrc/distributed but none of the 3 subdirectories in there looks right to me at first glance